cfg/SpeckleFilter.cfg

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020, Eurotec, Netherlands
```

Comment:
When adding copyright text to a file, could you add the text? If there is no copyright text in a file, then I think we can assume it falls under the `<license>BSD</license>` from the package.xml. Once you add "All rights reserved" here, it confuses things.

Reply:
Added a proper BSD license header